dashboard/src/components/dashboard.svelte

```diff
@@ -97,7 +97,12 @@
 	}
 
 	function onSearch() {
-		searchResults = devices.filter((d) => d.id?.includes(query));
+		if (query) {
+			if (query.includes('did:iota')) {
+				searchResults = devices.filter((d) => d.id?.includes(query));
+			} else searchResults = devices.filter((d) => d.name?.includes(query));
```

Comment:
please always add { } to conditions nevertheless it is just one statement inside